teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<p>Use f-strings?</p><blockquote>print(f"PubChem CID for Aspirin is:\n{cid}")</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>is that necessary here?</p>

Reply:
<p>Define necessary :) </p><p> </p><p>Two reasons that work best for me:</p><ul><li>For TeachOpenCADD we try to be as consistent as possible with writing our code. The other notebooks use f-strings, so should this notebook. Even more desirable, the style should be consistent within a notebook.</li><li>f-strings are the most elegant way of integrating variables in strings (though this might - in some examples - be a matter of taste and habit) - so teaching f-strings as the default is a goal for me as well</li></ul><p> </p><p>In your example above, using f-strings would homogenize these two lines of code:</p><p><code>raise IOError("Could not find matches for " + name)</code></p><p><code>print("PubChem CID for Aspirin is:\n", cid)</code></p><p> </p><p>Thus, no, it is not necessary in terms of the output but good practice in terms of consistency (and - for me - beauty) across our notebooks.</p><p> </p><p>I hope I could convince you :)</p>